test/views/helpers/LinkViewSpec.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.helpers
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import utils.ViewTest
+import views.html.templates.helpers.Link
+
+class LinkViewSpec extends ViewTest {
+
+  lazy val linkView: Link = app.injector.instanceOf[Link]
+
+  private val aTagSelector = "a"
+
+  "LinkView" when {
+    val link = "/contact/some-system"
+    val message = "Get help with this page"
+    val id = Some("tag-id")
+    val isExternal = true
+    val opensInNewTab = "(opens in new tab)"
+
+    "provided with " should {
```

Comment:
might be me being stupid, but provided with what?🙈

Reply:
You're not, it does not read well. I've made a change, let me know what you think.